tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java

```diff
@@ -246,7 +246,8 @@ public void endVariableRefList(int noArguments) {
     public void createAction(String actionName, boolean argsAvailable) {
         BLangModelBuilder.NameReference nameReference = nameReferenceStack.pop();
         if (processingActionInvocationStmt) {
-            modelBuilder.createActionInvocationStmt(null, null, nameReference, actionName, argsAvailable);
+            //  modelBuilder.createActionInvocationStmt(null, null, nameReference, actionName, argsAvailable);
```

Comment:
let's remove commented code

Reply:
This was fixed in this commit >> https://github.com/wso2/product-ei/pull/979/files#diff-e926d4a6b8bdff790cea82e4ff1ce334L295